chart/k8gb/templates/operator.yaml

```diff
@@ -111,7 +111,7 @@ spec:
             - name: NS1_ENABLED
               value: "true"
             {{ end }}
-            {{ if .Values.k8gb.exposeCoreDNS }}
+            {{ if eq "LoadBalancer" .Values.coredns.serviceType }}
             - name: COREDNS_EXPOSED
```

Comment:
@ytsarev, @k0da, is `COREDNS_EXPOSED` used for tests only? 
Name looks semantically confusing, as we always want associated CoreDNS to be exposed, regardless of serviceType.
Maybe it makes sense for tests to check if associated CoreDNS Service primitive is successfully created with "LoadBalancer" serviceType, and not created by default? In this case we don't have to wire this technical env variable.
Or at least it can be renamed to `COREDNS_EXTERNAL_LB` or similar.

Reply:
@somaritane no.
k8gb controller based on this setting:
if true:
- will check CoreDNS service for LoadBalancer IP
if false:
- will check GSLB object for ingress IPs (node ips)

Reply:
https://github.com/AbsaOSS/k8gb/blob/master/controllers/providers/dns/external.go#L64-L68

Reply:
@k0da I see, thanks, then I'd consider renaming it to `COREDNS_EXTERNAL_LB`.
Or, it can be renamed to `COREDNS_SERVICE_TYPE` and accept standard serviceType values ('LoadBalancer', 'ClusterIP'...)

Reply:
IMO we should either get rid of it or keep it boolean

Reply:
I agree,setting is confusing

Reply:
> IMO we should either get rid of it or keep it boolean

Question is, which settings we want controller logic to rely on: settings from helm release vs settings from real CoreDNS Service primitive spec.